src/traces/box/plot.js

```diff
@@ -94,12 +95,15 @@ function plotBoxAndWhiskers(sel, axes, trace, t, isStatic) {
 
         var posm0 = posAxis.l2p(lcenter - bdPos0 * nw) + bPosPxOffset;
         var posm1 = posAxis.l2p(lcenter + bdPos1 * nw) + bPosPxOffset;
-        var q1 = valAxis.c2p(d.q1, true);
-        var q3 = valAxis.c2p(d.q3, true);
+        var q1 = trace.sizemode === 'sd' ? valAxis.c2p(d.mean - 1 * d.sd, true) :
+                                           valAxis.c2p(d.q1, true);
```

Comment:
not a big deal but this (and a few below) can be simplified a little (is there a purpose to the `1*` or was that left over from the other version?):
```suggestion
        var sdmode = trace.sizemode === 'sd';
        var q1 = valAxis.c2p(sdmode ? d.mean - d.sd : d.q1, true);
```